src/api/connection.js

```diff
@@ -62,26 +64,26 @@ Connection_.prototype.handleResponse = function (responseCode, responseBody, que
 }
 
 Connection_.prototype.graphQLQuery = function (query, queryName) {
+  const response = this.fetchQuery(query)
   try {
-    var response = this.fetchQuery(query)
-    var responseCode = response.getResponseCode()
-    var responseBody = JSON.parse(response.getContentText())
+    const responseCode = response.getResponseCode()
+    const responseBody = JSON.parse(response.getContentText())
 
-    var logMessage = {
+    const logMessage = {
       type: 'RequestLog',
-      query: query,
-      queryName: queryName
```

Comment:
I would leave this to match the `type` way of construction. Furthermore, omitting the key when it's the name as the var name is a bad design decision and should not be used (strictly my opinion)

Reply:
Alright, I will disable the rule and do it this way. It is repetitive, but it looks better IMO too